mono/mini/driver.c

```diff
@@ -1689,6 +1689,18 @@ mono_get_version_info (void)
 
 static gboolean enable_debugging;
 
+void
+handle_stats_name (char *arg)
```

Comment:
This should be static, as it's not used outside of this file. I'd also make the argument name something more useful than `arg` if you intend to be passing only the method name.

Reply:
Oh, another nit: I'd also call it something like `set_stats_method_name` - handle is unnecessarily vague.